version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "16.4.1-SNAPSHOT"
+ThisBuild / version := "17.0.0-SNAPSHOT"
```

Comment:
Please set version to `17.0.0`, release binaries and then set it to snapshot again: `17.0.1-SNAPSHOT`